bookkeeper-server/src/test/java/org/apache/bookkeeper/server/http/TestHttpService.java

```diff
@@ -65,10 +68,11 @@
 import org.apache.bookkeeper.server.http.service.ClusterInfoService;
 import org.apache.bookkeeper.stats.NullStatsLogger;
 import org.apache.bookkeeper.test.BookKeeperClusterTestCase;
-import org.junit.Before;
-import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
```

Comment:
All the BookKeeper unit tests use JUnit, please do not import the testng dependency.